src/NumericalAlgorithms/Spectral/SwshCollocation.cpp

```diff
@@ -58,18 +58,19 @@ double Collocation<Representation>::phi(const size_t offset) const noexcept {
   return 2.0 * M_PI * ((offset % (2 * l_max_ + 1)) / (2.0 * l_max_ + 1.0));
 }
 
-namespace detail {
-// We use a `std::index_sequence` to loop over the function lambdas
-// for the cases we want to evaluate in order to only compute and store
-// a specific `l_max` that is requested, not all values up to
-// `collocation_maximum_l_max`. This cannot be done in the more traditional
-// way of either having a `std::vector` or `std::undordered_map` that is
-// `static` because only the construction phase is guaranteed to be thread-safe,
-// not the access. This would mean were we to use a `std::vector` or
-// `std::unordered_map` we would need to compute all values up to
-// `collocation_maximum_l_max` immediately. By doing the loop over `Is` we
-// are able to generate `sizeof...(Is)` different static objects and thus each
-// one can be constructed only when it is needed and in a thread-safe manner.
+namespace {
+// We use a `std::index_sequence` to loop over all the l's up to l_max. Each
```

Comment:
I feel describing the index sequence process as a loop is no longer quite accurate.